src/components/Reports/ReportEdit.tsx

```diff
@@ -141,8 +144,34 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
+  const hasEmptyRequiredFields = (iReport: Report) => {
+    const emptySeedbankFields = iReport.seedBanks?.some((sb) => {
+      return !sb.buildStartedDate || !sb.buildCompletedDate || !sb.operationStartedDate;
+    });
+    const emptyNurseryFields = iReport.nurseries?.some((nursery) => {
+      return (
+        !nursery.buildStartedDate || !nursery.buildCompletedDate || !nursery.operationStartedDate || !nursery.capacity
+      );
+    });
+    const emptyPlantingSitesFields = iReport.plantingSites?.some((plantingSite) => {
+      return (
+        !plantingSite.totalPlantingSiteArea ||
+        !plantingSite.totalPlantedArea ||
+        !plantingSite.totalTreesPlanted ||
+        !plantingSite.totalPlantsPlanted ||
+        !plantingSite.mortalityRate
+      );
+    });
+    return emptySeedbankFields || emptyNurseryFields || emptyPlantingSitesFields;
+  };
   const submitReport = async () => {
     if (report) {
+      if (hasEmptyRequiredFields(report)) {
+        setConfirmSubmitDialogOpen(false);
+        setValidateFields(true);
+        handleBack();
```

Comment:
Actually, would you mind doing
```typescript
if (showAnnual) {
  handleBack();
}
```
here? My thought being that if they are already on the quarterly page a Save might not be intended.

Reply:
Oh, you are right!